index.html

```diff
@@ -1 +1,55 @@
+<h1>Email Markup Consortium</h1>
 
+<p>Working to improve the user experience, accessibility, performance,
+consistency, and reliability of email markup</p>
+
+<hr>
+
+<h2>Our Mission</h2>
+
+<p>The Email Markup Consortium (EMC) is a community-led group of industry professionals working to improve the email experience for everyone. Recipients should be able to easily access the content of every email sent to them, regardless of their device, email client, assistive technology or internet speed. The EMC collaborates with developers, creation and sending tools, and email clients to drive this vision forward at every step of the email journey.
+
+<p>Senders, recipients, and email clients share the common goals of improving user experience, accessibility, performance, consistency, and reliability. However, currently there is a lack of cooperation in trying to achieve these goals leading to inconsistent results. By working together on agreed approaches we can save everyone time, effort, and money while delivering better results to the recipients.</p>
+
+<h2>How You Can Help</h2>
+
+<ol>
+  <li>PLACEHOLDER: Share on Twitter, LinkedIn, etc (links with share URLs?)</li>
+  <li>PLACEHOLDER: Join us (How? Link to form, GitHub repo)</li>
```

Comment:
Need to write some compelling CTAs here.

Reply:
I find the "how to use the community" section on https://workonclimate.org/ is a nice way to do it, but it may be a bit early for us to categorise our links like that.

They could be (not final wording):

- Join as a member (membership form)
- Join the discussion/community - contribution with no commitment ([GitHub discussions](https://github.com/email-markup-consortium/email-markup-consortium/discussions) is a free forum; we can perhaps use it more?)
- Support/sponsorship? Perhaps this is a future one
- Spread the word / share on social media

Reply:
oh I like how their landing page is set up! Maybe starting with the first two list items as CTAs is good enough to launch?